zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/DefaultExporterResourceProvider.java

```diff
@@ -78,45 +83,49 @@ public void init(final ExporterConfiguration configuration) {
         ConnectionTypes.from(configuration.getConnect().getType())
             .equals(ConnectionTypes.ELASTICSEARCH);
 
+    final var templateDescriptors =
+        List.of(
+            entry(ListViewTemplate.class, new ListViewTemplate(globalPrefix, isElasticsearch)),
+            entry(VariableTemplate.class, new VariableTemplate(globalPrefix, isElasticsearch)),
+            entry(
+                PostImporterQueueTemplate.class,
+                new PostImporterQueueTemplate(globalPrefix, isElasticsearch)),
+            entry(
+                FlowNodeInstanceTemplate.class,
+                new FlowNodeInstanceTemplate(globalPrefix, isElasticsearch)),
+            entry(IncidentTemplate.class, new IncidentTemplate(globalPrefix, isElasticsearch)),
+            entry(
+                SequenceFlowTemplate.class,
+                new SequenceFlowTemplate(globalPrefix, isElasticsearch)),
+            entry(
+                DecisionInstanceTemplate.class,
+                new DecisionInstanceTemplate(globalPrefix, isElasticsearch)),
+            entry(EventTemplate.class, new EventTemplate(globalPrefix, isElasticsearch)),
+            entry(TaskTemplate.class, new TaskTemplate(globalPrefix, isElasticsearch)));
+
+    templateDescriptors.forEach(entry -> entry.getValue().setIsGlobalPrefix(true));
```

Comment:
🔧 the suggestion above about the builder-style with can help get rid of this iteration, and then the collection to a map later and make it more explicit. (if you don't want to include the field on all constructors)